app/views/events/index.haml

```diff
@@ -0,0 +1,44 @@
+%h2 My events
+- if current_user.events.any?
+  %table.table.table-striped.user-events
```

Comment:
table is hard to style properly for smaller devices. Maybe try to use list selector `ul` and some fancy bootstrap classes?